extensions/wikia/EmbeddableDiscussions/models/DiscussionsThreadModel.class.php

```diff
@@ -90,14 +100,14 @@ private function formatData( $rawData, $showLatest ) {
 		return $data;
 	}
 
-	public function getData( $showLatest, $limit ) {
+	public function getData( $showLatest, $limit, $category ) {
 		$memcKey = wfMemcKey( __METHOD__, self::MCACHE_VER );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)